personhood/phapp/test.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+DBG_TEST=1
+DBG_SRV=2
+DBG_BA=2
+DBG_PH=2
+
+NBR_SERVERS=4
+NBR_SERVERS_GROUP=3
+
+ZERO_KEY=0000000000000000000000000000000000000000000000000000000000000000
+
+. "$(go env GOPATH)/src/github.com/dedis/cothority/libtest.sh"
+
+main(){
+    startTest
+    buildConode github.com/dedis/cothority/byzcoin github.com/dedis/cothority/byzcoin/contracts github.com/dedis/cothority/personhood
+    build $APPDIR/../../byzcoin/bcadmin
+    run testSpawner
+    run testWipe
+    run testRegister
+    stopTest
+}
+
+testSpawner(){
+  rm -f config/*
+  runCoBG 1 2 3
```

Comment:
might be worth to have some sleep here and the other same lines to let the conodes start. It seems slow sometimes on Jenkins which makes the tests fail.

Reply:
That's an error in `libtest.sh` then, as it's supposed to wait for the conodes to have started up:

https://github.com/dedis/cothority/blob/master/libtest.sh#L370

BTW: `sleep 10` there seems really long. It should correctly `ping` the servers or something like that.

Reply:
Fixed by #1741